starknet/class.go

```diff
@@ -47,12 +47,13 @@ type ClassDefinition struct {
 }
 
 type CompiledClass struct {
-	Prime           string          `json:"prime"`
-	Bytecode        []*felt.Felt    `json:"bytecode"`
-	Hints           json.RawMessage `json:"hints"`
-	PythonicHints   json.RawMessage `json:"pythonic_hints"`
-	CompilerVersion string          `json:"compiler_version"`
-	EntryPoints     struct {
+	Prime                  string          `json:"prime"`
+	Bytecode               []*felt.Felt    `json:"bytecode"`
+	Hints                  json.RawMessage `json:"hints"`
+	PythonicHints          json.RawMessage `json:"pythonic_hints"`
+	CompilerVersion        string          `json:"compiler_version"`
+	BytecodeSegmentLengths []interface{}   `json:"bytecode_segment_lengths"`
```

Comment:
can we add a testcase with sample data from integration network?
also, let's make this a concrete type instead of `[]any`